core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/ProjectImpl.java

```diff
@@ -61,4 +154,317 @@ public ProjectImpl(User user) {
     setSecurityProfile(new SecurityProfile(user));
   }
 
+  @Override
+  public Date getCreationDate() {
+    return creationDate;
+  }
+
+  @Override
+  public String getDescription() {
+    return description;
+  }
+
+  @Override
+  public String getName() {
+    return name;
+  }
+
+  @Override
+  public String getAlias() {
+    return alias;
+  }
+
+  @Override
+  public String getShortName() {
+    return shortName;
+  }
+
+  @Override
+  public void setShortName(String shortName) {
+    this.shortName = shortName;
+  }
+
+  @Override
+  public long getId() {
+    return projectId;
+  }
+
+  @Override
+  public void setId(long id) {
+    this.projectId = id;
+  }
+
+  @Override
+  @Deprecated
+  public Long getProjectId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getId" on line 188. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)